### DISCUSSION TITLE
Photoprism LXC outdated libheif version

### DISCUSSION THREAD

**Discussion by dsiebel** [tteck/Proxmox#3344] [General]
### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

The Photoprism LXC container comes with an outdated version of `libheif`: `1.15.1`.
The install script contains the download of the pre-bundles `libheif` tar, provided by Photoprism, but it doesn't not seem to be correctly installed.

The install script uses

```sh
wget -q -cO - https://dl.photoprism.app/dist/libheif/libheif-bookworm-amd64-v1.17.1.tar.gz | tar -xzf - -C /usr/local --strip-components=1
```

to "install" to /usr/local.

The [README](https://dl.photoprism.app/dist/libheif/README.html) however recommends installing via the provided [install scipt](https://raw.githubusercontent.com/photoprism/photoprism/develop/scripts/dist/install-libheif.sh):

```sh
bash <(curl -s https://raw.githubusercontent.com/photoprism/photoprism/develop/scripts/dist/install-libheif.sh)
```

This successfully installed `libheif-v1.17.1` in my Photoprism LXC and I can finally import HEIC images again since switching from custom Docker setup 😅 

Happy to provide a PR with the changes, if you agree this should be adjusted.
Thanks for your amazing work!

### What settings are you currently utilizing?

Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### Please provide detailed steps to reproduce the issue.

- open shell to container
- run `heif-convert --version`
- should print `1.17.1` but instead prints `1.15.1` (installed from debian sources)

**Comment by dsiebel** [comment]
Had a closer look at the aformentioned install script. It doesn't seem to do things that much different (wget). However, the install scripts additionally links the runtime bindings using `ldconfig`:
https://github.com/photoprism/photoprism/blob/develop/scripts/dist/install-libheif.sh#L81

Maybe that's all that's missing?

**Comment by tteck** [comment]
This is from the script
https://github.com/tteck/Proxmox/blob/186f186e8b6c8d706adc8a110646f93147e35448/install/photoprism-install.sh#L36-L43

**Reply by dsiebel** [reply]
Whoops, my bad. I read the script like three times and completely missed it.

**Comment by tteck** [comment]
`ldconfig` is in the script.

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/blob/186f186e8b6c8d706adc8a110646f93147e35448/install/photoprism-install.sh#L22 could this be causing the issue?

**Reply by dsiebel** [reply]
It could be. I didn't uninstall it though, just ran the install script as described in the README.

**Reply by dsiebel** [reply]
The only other difference I see right now is `--strip-components=1`.
